src/Scrapers/CFScraper.py

```diff
@@ -12,7 +12,12 @@
 class CFScraper(Scraper):
     def __init__(self, url) -> None:
         super().__init__(url)
-        response = requests.get(url)
+        # Because request only makes http request calls, change our https link to http
+        protocol , link = url.split(':')
+        protocol = 'http'
+        url = protocol + ':'+link
+        proxy = Scraper(url).get_proxy()
+        response = requests.get(url,proxies = proxy,verify = False)
         html_content = response.content
         self.soup = BeautifulSoup(html_content, 'lxml')
```

Comment:
We can keep this entire code in the Scraper base class as it would be same for all type of scrapers. This was the thing I was pointing out in earlier PR.